C#/cs01/04.cs

```diff
@@ -9,9 +9,20 @@ namespace Brainchild.Curriculum.Solutions
     Fizz Buzz, 16, 17, Fizz, 19, Buzz, Fizz, 22, 23, Fizz, Buzz, 26, 
     Fizz, 28, 29, Fizz Buzz, 31, 32, Fizz, 34, Buzz, Fizz, ...
     */
-    public class FizzBuzzV2
+    public class FizzBuzzV2:FizzBuzzPrinter
     {
        //this class should reuse existing FizzBuzzPrinter from C#\cs01\01.cs
        //HINT:YOu have to use inheritance https://www.w3schools.com/cs/cs_inheritance.asp
+       public string[] printNumbersV2(){
+           string[] result=PrintNumbers(100);
```

Comment:
We ae calling the same logic all over. we are not reusing the functions here but actually duplicting the logic.think about how we can reuse the function